flow360/component/flow360_params/params_base.py

```diff
@@ -289,6 +299,28 @@ def handle_deprecated_aliases(cls, values):
                 values = cls._handle_depracated_alias(values, deprecated_alias)
         return values
 
+    # pylint: disable=no-self-argument
+    @pd.root_validator(pre=True)
+    def handle_conflicting_fields(cls, values):
+        """
+        root validator to handle deprecated aliases
+        """
+        if cls.Config.conflicting_fields:
+            for conflicting_field in cls.Config.conflicting_fields:
+                values = cls._handle_conflicting_fields(values, conflicting_field)
+        return values
+
+    @classmethod
+    def _handle_conflicting_fields(cls, values, conflicting_field: Conflicts = None):
+        conflicting_field1_value = values.get(conflicting_field.field1, None)
+        conflicting_field2_value = values.get(conflicting_field.field2, None)
+        if conflicting_field1_value is not None and conflicting_field2_value is not None:
+            raise ValueError(
+                f"{conflicting_field.field1} and {conflicting_field.field2} cannot be specified at the same time."
+            )
```

Comment:
make sure that it works for both name and alias name

Reply:
Done. Just note that in the future we should not add default value for conflicting fields otherwise `to_file_from_file_test()` will fail.